src/views/Predictions/components/RoundCard/SetPositionCard.tsx

```diff
@@ -38,6 +39,10 @@ interface SetPositionCardProps {
   onSuccess: (decimalValue: BigNumber, hash: string) => Promise<void>
 }
 
+// /!\ TEMPORARY /!\
+// Set default gasPrice (6 gwei) when calling BetBull/BetBear before new contract is released fixing this 'issue'.
+const gasPrice = new BigNumber(6).times(BIG_TEN.pow(BIG_NINE)).toString()
```

Comment:
I realise you'll probably be on top of this Kai when the change can be made, but we often add the text 'TODO' in the comment so that temporary code / work to be done can be easily searched for and found - can you add?

Reply:
Done in cd3d4c4b9c3cdcb0b1e85e180652ffb56d965651